Library/Formula/librdkafka.rb

```diff
@@ -0,0 +1,10 @@
+class Librdkafka < Formula
+  homepage "https://github.com/edenhill/librdkafka"
+  url "http://github.com/edenhill/librdkafka/archive/0.8.6.tar.gz"
+  sha256 "184080e3898b80b3f7c1398c50787f8edb326b8271017a5f8000ef9a660e1a4f"
+
+  def install
+    system "./configure", "--disable-dependency-tracking", "--prefix=#{prefix}"
+    system "make", "install"
+  end
+end
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!